test/observations.js

```diff
@@ -61,25 +61,6 @@ test('observations: create + delete', function (t) {
   })
 })
 
-test('observations: create invalid', function (t) {
```

Comment:
bonus points if you make this a test that sends bad json, to check that that code works